datadog_lambda/tracing.py

```diff
@@ -248,6 +250,8 @@ def create_function_execution_span(
             "function_version": function_version,
             "request_id": context.aws_request_id,
             "resource_names": context.function_name,
+            "datadog_lambda_py_version":  lambda_version,
```

Comment:
Now looking at this, I feel the key might be a little bit too verbose, probably we could leave them as `datadog_lambda` and `dd_trace`, because `py` is something we already know from the runtime tag, and version seems implicit when you look at the value.

Reply:
fixed!